resources/lib/apihelper.py

```diff
@@ -363,8 +450,8 @@ def get_latest_episode(self, program):
         video = dict(listitem=video_item, video_id=episode.get('videoId'), publication_id=episode.get('publicationId'))
         return video
 
-    def get_episodes(self, program=None, season=None, category=None, feature=None, programtype=None, keywords=None, whatson_id=None, video_id=None,
-                     video_url=None, page=None, use_favorites=False, variety=None, cache_file=None):
+    def get_episodes(self, program=None, season=None, episodes=None, category=None, feature=None, programtype=None, keywords=None,
```

Comment:
This `get_episodes()` is becoming one big filtering method :-)
It works well and is quite easy to read, but it somehow looks wrong as an interface.
Not sure what the solution is, everything I can think of seems worse.

Reply:
I think it's quite normal that we use a lot of parameters when communicating with an extensive API like the VRT Search API. We can combine data using objects, but I didn't take the time yet to think about a better data design. Metadata collection and processing is definitely something that can improve.